manifests/resource-topology-exporter-ds.yaml

```diff
@@ -56,6 +56,16 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: spec.nodeName
+        - name: REFERENCE_NAMESPACE
```

Comment:
Don't we need ${REFERENCE_NAMESPACE}, ${REFERENCE_POD_NAME} and ${REFERENCE_CONTAINER_NAME} here?

Reply:
no, because this is the name of the env vars being created

Reply:
Ah, okay. Sorry missed that these are the environment variable KEY values.